contracts/core/src/libraries/ConfLevel.sol

```diff
@@ -5,24 +5,15 @@ pragma solidity ^0.8.12;
  * @title ConfLevel
  * @notice XMsg confirmation levels. Matches ConfLevels in lib/xchain/types.go
  * @dev We prefer explicit constants over Enums, because we want uint8 values to start at 1, not 0, as they do in
- *      lib/xchain/types.go, such that 0 can represent "unset".
+ *      lib/xchain/types.go, such that 0 can represent "unset". Note only latest and finalized levels are supported
+ *      on-chain.
  */
 library ConfLevel {
     /**
      * @notice XMsg confirmation level "latest", last byte of xmsg.shardId.
      */
     uint8 internal constant Latest = 1;
```

Comment:
can we maybe use an enum for this?

Reply:
Enum are zero indexed and we start with one. And they always increment montonically. So the translation to lib/xchain/types.go values is just a little annoying. I'd rather just set constants explicitly